### PR TITLE
Remove batch action checkboxes in ModelListType list

### LINKED ISSUE #4521: Batch action checkboxes in ModelListType list
#### Sonata packages
<details>

```bash
$ composer show sonata-project/*
sonata-project/admin-bundle              3.19.0             The missing Symfony Admin Generator
sonata-project/block-bundle              3.3.1              Symfony SonataBlockBundle
sonata-project/cache                     1.0.7              Cache library
sonata-project/core-bundle               3.4.0              Symfony SonataCoreBundle
sonata-project/datagrid-bundle           2.2.1              Symfony SonataDatagridBundle
sonata-project/doctrine-extensions       1.0.2              Doctrine2 behavioral extensions
sonata-project/doctrine-orm-admin-bundle 3.1.5              Symfony Sonata / Integrate Doctrine ORM into the SonataAdminBundle
sonata-project/easy-extends-bundle       2.1.10             Symfony SonataEasyExtendsBundle
sonata-project/exporter                  1.7.1              Lightweight Exporter library
sonata-project/media-bundle              3.5.1              Symfony SonataMediaBundle
sonata-project/notification-bundle       3.1.0              Symfony SonataNotificationBundle
sonata-project/user-bundle               dev-master b375d57 Symfony SonataUserBundle
```
</details>

## Subject

In `ModelListType` it is possible to select an item from the list and there are checkboxes in this list,  but the batch actions are hidden and  there's nothing you can do with those checkboxes.

batch_action_checkboxes

same issue #2979 

## Expected results
No batch action checkboxes
## Actual results
Column with checkboxes

### PR DESCRIPTION
**Pull request by jenyak**
I am targeting this branch, because of a bug fix.

Closes #4521 

## Changelog

```markdown
### Fixed
- Remove batch action checkboxes in ModelListType list
```

### CODE REVIEW

File: Resources/views/CRUD/base_list.html.twig [hunk 1]
```diff
@@ -35,7 +35,8 @@ file that was distributed with this source code.
 {% block list_table %}
     <div class="col-xs-12 col-md-12">
         {% set batchactions = admin.batchactions %}
-        {% if admin.hasRoute('batch') and batchactions|length %}
+        {% set show_batchactions = batchactions|length and admin.hasRoute('batch') and not app.request.isXmlHttpRequest %}
```
Comment (jlamur) [inline Resources/views/CRUD/base_list.html.twig]: I think `batchactions is not empty` should be prefered over `batchactions|length`:

```diff
- {% set show_batchactions = batchactions|length and admin.hasRoute('batch') and not app.request.isXmlHttpRequest %}
+ {% set show_batchactions = batchactions is not empty and admin.hasRoute('batch') and not app.request.isXmlHttpRequest %}
```

Comment (jlamur) [inline Resources/views/CRUD/base_list.html.twig]: Nothing more than readability.

**Comment by OskarStark** [comment]
Hey @jenyak 👋 

Would you like to finish this PR? If yes, please address @jlamur 's comments, thank you!

**Comment by core23** [comment]
This PR looks dead. Feel free to reopen if you will continue working on this.

**Pull request closed without merging**